### PR TITLE
Demo app cleanup

### PR DESCRIPTION
**Pull request by goergisn**
## Summary
- Removing unused Demo App assets

### CODE REVIEW

**Review by nauaros — DISMISSED** [review]
Thank you 🙏

**Review by erenbesel — DISMISSED** [review]
👏

File: Demo/SwiftUI/ComponentsViewModel.swift [hunk 1]
```diff
@@ -112,7 +112,7 @@ internal final class ComponentsViewModel: ObservableObject, Identifiable {
                 ),
                 ComponentsItem(
                     title: "Instant/Redirect Payment",
-                    subtitle: "e.g. PayPal, Alipay, ...",
+                    subtitle: "e.g. Ideal, PayPal, Alipay, ...",
```
Comment (descorp) [inline Demo/SwiftUI/ComponentsViewModel.swift]: Embrace modernity 😄

**Review by descorp — DISMISSED** [review]

**Review by atmamont — DISMISSED** [review]
👍

**Review by erenbesel — APPROVED** [review]

**Pull request merged**